include/aikido/constraint/TSR.hpp

```diff
@@ -44,7 +44,8 @@ class TSR : public Sampleable,
       const Eigen::Isometry3d& _T0_w = Eigen::Isometry3d::Identity(),
       const Eigen::Matrix<double, 6, 2>& _Bw =
           Eigen::Matrix<double, 6, 2>::Zero(),
-      const Eigen::Isometry3d& _Tw_e = Eigen::Isometry3d::Identity());
+      const Eigen::Isometry3d& _Tw_e = Eigen::Isometry3d::Identity(),
+      const double& _satisfiableTolerance = 1e-6);
```

Comment:
Nit: Passing const reference for primitive type is pointless. Please change to `double`.